draft-ietf-scitt-architecture.md

```diff
@@ -201,7 +201,7 @@ Artifact:
 
 Auditor:
 
-: an entity that checks the correctness and consistency of all Transparent Statements registered by a Transparency Service (a subset of potential Transparent Statement Consumers).
+: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service (to a subset of potential Transparent Statement Consumers).
```

Comment:
The directionality of this, feels difficult...

I agree auditor checks transparent statements... not sure the parenthetical is helpful.

Reply:
ok! Agree with the statement, will change